hedera-mirror-rest/api/v1/openapi.yml

```
@@ -2036,6 +2059,67 @@ components:
       type: array
       items:
         $ref: '#/components/schemas/NetworkFee'
+    NetworkStake:
+      type: object
+      required:
+        - max_staking_reward_rate_per_hbar
+        - node_reward_fee_fraction
+        - stake_total
+        - staking_period
+        - staking_period_duration
+        - staking_periods_stored
+        - staking_reward_fee_fraction
+        - staking_reward_rate
+        - staking_start_threshold
+      properties:
+        max_staking_reward_rate_per_hbar:
+          description: The maximum reward rate, in tinybars per whole hbar, that any account can receive in a day
+          type: integer
+          format: int64
+        node_reward_fee_fraction:
+          description: The fraction of the network and service fees paid to the node reward account 0.0.801
+          type: number
+          format: int64
+        stake_total:
+          description: The total amount staked to the network in tinybars the start of the current staking period
+          type: number
+          format: int64
```

number does not [support](https://swagger.io/docs/specification/data-models/data-types/#numbers) a format of int64. Use integer everywhere instead.
```suggestion
          type: integer
          format: int64
```

---

Thanks I have changed it to integer for all the new fields.